test/test_tree_2d_euler.jl

```diff
@@ -400,6 +400,7 @@ end
 end
 
 @trixi_testset "elixir_euler_sedov_blast_wave_sc_subcell.jl" begin
+    rm("out/deviations.txt", force = true)
```

Comment:
```suggestion
    rm(joinpath("out", "deviations.txt"), force = true)
```
OS agnostic

Reply:
Same applies to rest of file

Reply:
Thanks. See [c196ec3](https://github.com/trixi-framework/Trixi.jl/pull/1824/commits/c196ec35ec03e675d87f6c9b114a52652659d0f2).